lib/collection/event-list.js

```diff
@@ -40,7 +40,7 @@ _.assign(EventList.prototype, /** @lends EventList.prototype */ {
             // array we are compiling for return
             (parent !== this.__parent) && EventList.isEventList(parent.events) &&
                 (parentEvents = parent.events.listenersOwn(name)) && parentEvents.length &&
-                all.push.apply(all, parentEvents);
+                all.push.apply(all, parentEvents); // eslint-disable-line prefer-spread
```

Comment:
I can go ahead with the merge for now, but need to know why this was not causing problems before.